content/en/lotus/manage/chain-management.md

```diff
@@ -31,31 +31,18 @@ We recommend most users perform the initial node sync from a lightweight snapsho
 These lightweight state snapshots **do not contain any message receipts**. To get message receipts, you need to sync your Lotus node from the genesis block without using any of these snapshots.
 {{< /alert >}}
 
-1. Download the most recent lightweight snapshot and its checksum:
+1. Download the most recent lightweight snapshot:
 
     a. For **mainnet**, command always contains the latest snapshot available for mainnet:
 
     ```shell
-    curl -sI https://snapshots.mainnet.filops.net/minimal/latest | perl -ne '/x-amz-website-redirect-location:\s(.+)\.car/ && print "$1.sha256sum\n$1.car"' | xargs wget
+    wget https://snapshots.mainnet.filops.net/minimal/latest
```

Comment:
Sadly a bare `wget` here is a problem, due to it taking on the non-descript name `latest` locally. It seems like `aria2c -x5` does the right thing, and is more appropriate anyway ( please test ‼️ )

Reply:
Tested - Resolved. 
We might need to add `aria2c` as dependency - or tips alert.